sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/SpannerAccessor.java

```diff
@@ -177,7 +177,14 @@ static SpannerOptions buildSpannerOptions(SpannerConfig spannerConfig) {
               .setInitialRpcTimeout(org.threeten.bp.Duration.ofMinutes(120))
               .setMaxRpcTimeout(org.threeten.bp.Duration.ofMinutes(120))
               .setTotalTimeout(org.threeten.bp.Duration.ofMinutes(120))
+              .setRpcTimeoutMultiplier(1.0)
+              .setInitialRetryDelay(org.threeten.bp.Duration.ofSeconds(2))
```

Comment:
setInitialRetryDelay, setMaxRetryDelay are marked as ObsoleteApi by Spanner client. Consider move to use setInitialRetryDelayDuration with java.time.Duration parameters. Could take the chance to also fix the parameters around (setInitialRpcTimeout, setMaxRpcTimeout, setTotalTimeout)

Reply:
Done. changed all the parameters in this file to use the non-deprecated version